Packs/Tenable_io/Integrations/Tenable_io/Tenable_io_test.py

```diff
@@ -40,28 +40,137 @@
+        },
+        "recasted_count": 0,
+        "reference_information": [
+            {
+                "name": "cve",
+                "url": "http://web.nvd.nist.gov/view/vuln/detail?vulnId=",
+                "values": [
+                    "CVE-1999-0524"
+                ]
+            },
+            {
+                "name": "cwe",
+                "url": "notaurl",
+                "values": [
+                    "200"
+                ]
+            },
+            {
+                "name": "osvdb",
+                "values": [
+                    "94"
+                ]
+            }
+        ],
+        "risk_information": {
+            "cvss3_base_score": None,
+            "cvss3_temporal_score": None,
+            "cvss3_temporal_vector": None,
+            "cvss3_vector": None,
+            "cvss_base_score": None,
+            "cvss_temporal_score": None,
+            "cvss_temporal_vector": None,
+            "cvss_vector": None,
+            "risk_factor": "None",
+            "stig_severity": None
+        },
+        "see_also": [],
+        "severity": 0,
+        "solution": "Filter out the ICMP timestamp requests (13), and the outgoing ICMP timestamp replies (14).",
+        "synopsis": "It is possible to determine the exact time set on the remote host.",
+        "vuln_count": 20844,
+        "vulnerability_information": {
+            "asset_inventory": None,
+            "cpe": None,
+            "default_account": None,
+            "exploit_available": None,
+            "exploit_frameworks": [],
+            "exploitability_ease": None,
+            "exploited_by_malware": None,
+            "exploited_by_nessus": None,
+            "in_the_news": None,
+            "malware": None,
+            "patch_publication_date": None,
+            "unsupported_by_vendor": None,
+            "vulnerability_publication_date": "1995-01-01T00:00:00Z"
+        }
+    }
+}
 
 def mock_demisto(mocker, mock_args):
     mocker.patch.object(demisto, 'params', return_value=MOCK_PARAMS)
     mocker.patch.object(demisto, 'args', return_value=mock_args)
     mocker.patch.object(demisto, 'results')
 
+def test_get_scans_command(mocker, requests_mock):
+    mock_demisto(mocker, {'folderId': '34', 'examplelastModificationDate': '2020-11-01' })
+    requests_mock.get(MOCK_PARAMS['url'] + 'scans/?folder_id=34', json=MOCK_GET_SCANS_REQUEST)
+    from Tenable_io import get_scans_command
+    get_scans_command()
 
-def test_get_scan_status(mocker, requests_mock):
-    mock_demisto(mocker, {'scanId': '25'})
+def test_launch_scan_command(mocker, requests_mock):
+    mock_demisto(mocker, {'scanId': '25', 'scanTargets': '10.0.0.1'})
     requests_mock.get(MOCK_PARAMS['url'] + 'scans/25', json={'info': {'status': 'canceled'}})
+    requests_mock.post(MOCK_PARAMS['url'] + 'scans/25/launch?alt_targets=10.0.0.1', json={'info': {'status': 'canceled'}})
+    from Tenable_io import launch_scan_command
+    launch_scan_command()
```

Comment:
Missing `assert`

Reply:
Update 02.23.2021 1034 HRS EST:
Not Started